XRTK-Core/Packages/com.xrtk.core/Definitions/Controllers/Hands/HandData.cs

```diff
@@ -13,7 +14,12 @@ namespace XRTK.Providers.Controllers.Hands
     public class HandData
     {
         /// <summary>
-        /// Timestamp of hand data, as FileTime, e.g. DateTime.UtcNow.ToFileTime().
+        /// Gets the total count of joints the hand data supports.
+        /// </summary>
+        public static readonly int JointCount = Enum.GetNames(typeof(TrackedHandJoint)).Length;
```

Comment:
Cool

Reply:
I thought it made more sense here than in the controller class.